gazelle/internal/swiftpkg/swift_file_info_test.go

```diff
@@ -73,25 +73,63 @@ func TestNewSwiftFileInfoFromSrc(t *testing.T) {
 	})
 }
 
-func TestDirSuffixes(t *testing.T) {
-	actual := swiftpkg.TestDirSuffixes.IsUnderDirWithSuffix("Sources/Foo/Bar.swift")
+func TestDirectoryPathSuffixes(t *testing.T) {
+	actual := swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Sources/Foo/Bar.swift")
 	assert.False(t, actual)
 
-	actual = swiftpkg.TestDirSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Bar.swift")
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("sources/foo/bar.swift")
+	assert.False(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/foo_tests/bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Chicken/Bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/foo_tests/chicken/bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/Bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/bar.swift")
+	assert.True(t, actual)
+}
+
+func TestFilePathSuffixes(t *testing.T) {
+	actual := swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Sources/Foo/Bar.swift")
+	assert.False(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("sources/foo/bar.swift")
+	assert.False(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/foo_tests/bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Chicken/Bar.swift")
+	assert.True(t, actual)
+
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/foo_tests/chicken/bar.swift")
 	assert.True(t, actual)
 
-	actual = swiftpkg.TestDirSuffixes.IsUnderDirWithSuffix("Tests/FooTests/Chicken/Bar.swift")
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("Tests/Bar.swift")
 	assert.True(t, actual)
 
-	actual = swiftpkg.TestDirSuffixes.IsUnderDirWithSuffix("Tests/Bar.swift")
+	actual = swiftpkg.TestDirectoryPathSuffixes.IsUnderDirWithSuffix("tests/bar.swift")
 	assert.True(t, actual)
 }
 
 func TestSwiftFileInfos(t *testing.T) {
 	tests := []struct {
-	    msg string
-	    fileInfos swiftpkg.SwiftFileInfos
-	    exp bool
+		msg       string
+		fileInfos swiftpkg.SwiftFileInfos
+		exp       bool
```

Comment:
Is this indent correct? It looks like it indented too far to the right.

Reply:
Oh somehow it switched this to tabs instead of spaces -- I'll find and replace the tabs.

Reply:
Actually most of the file was already using tabs and this was the auto-formatter correcting this code.
Github renders tabs as 8 spaces, but VSCode defaults to 4 (you can change it).